opusdec.profile

```diff
@@ -0,0 +1,25 @@
+blacklist /usr/local/bin
+blacklist /usr/bin
+blacklist /bin
+blacklist /sbin
+blacklist /boot
+blacklist /media
+blacklist /mnt
+blacklist /opt
+blacklist /var
+blacklist /tmp
+
+private-bin opusdec
+
+noexec /home
+noexec /tmp
+
+shell none
+seccomp
+caps.drop all
+net none
+noroot
+nosound
+nogroups
+nonewprivs
+ipc-namespace
```

Comment:
you might want to protect writes to the filesystem with something like:

```
read-write /xxx
read-only /
```
or
```
whitelist /tmp/work
read-only /
```

or even:
```
overlay-tmpfs
```

of course, if ever possible to add:
```
seccomp.drop=execve
```
that would be nice too - otherwise its definitely a good start/much better that not having it